m47/PresentationConnectionAvailableEvent_Constructor.md

```diff
@@ -0,0 +1,28 @@
+# PresentationConnectionAvailableEvent()
+
+The `PresentationConnectionAvailableInit()` constructor creates a new [PresentationConnectionAvailableEvent ](PresentationConnectionAvailableEvent.md) is fired on a [PresentationRequest](PresentationRequest.md) when a connection associated with the object is created.
+
+## Syntax
+
+`var PresentationConnectionAvailableEvent  = new PresentationConnectionAvailableInit(type, PresentationConnectionAvailableInit)`
+
+### Returns
+
+An instance of the [PresentationConnectionAvailableEvent ](PresentationConnectionAvailableEvent.md) interface.
+
+### Parameters
+
+<dl>
+  <dt>type</dt>
+  <dd>TBD</dd>
+  <dt>PresentationConnectionAvailableEvent </dt>
+  <dd>TBD</dd>
+</dl>
+
+## Example
+
+TBD
+
+## Specifications
+
+<http://w3c.github.io/presentation-api/#interface-presentationsessionconnectevent>
```

Comment:
I'm not sure what you're asking. I've removed the word session in all the text except the spec link. The spec link can't change until the spec does.